ONSdigital/eq-survey-runner
# fix radio/checkbox styling

### What is the context of this PR?

Radio button and checkbox styling was using browser defaults instead of our CSS, this PR fixes it.
### How to review
1. Start star Wars survey.
2. Confirm radio buttons and checkboxes are styled appropriately:

<img width="694" alt="screenshot 2016-10-05 13 29 01" src="https://cloud.githubusercontent.com/assets/930398/19113023/b9326744-8aff-11e6-81fd-085e069762ed.png">
